src/ensembl/src/content/app/species-selector/components/species-commit-button/SpeciesCommitButton.test.tsx

```diff
@@ -30,11 +35,26 @@ jest.mock(
     }))
 );
 
-const onCommit = jest.fn();
-const defaultProps = {
-  currentSpecies: createSelectedSpecies() as CurrentItem,
-  disabled: false,
-  onCommit
+const committedHuman: ReturnType<typeof createSelectedSpecies> = {
+  ...createSelectedSpecies(),
+  genome_id: 'human'
+};
+
+const defaultReduxState = {
+  speciesSelector: {
+    currentItem: committedHuman
+  }
+};
+
+const mockStore = configureMockStore([thunk]);
+
+const renderComponent = (state?: unknown) => {
```

Comment:
I see state typed as any in PopularSpeciesButton.test where as here it is unknown. I dont know which is better.

Reply:
Generally, `unknown` is better. In the case of these tests though, both are equally bad 😄 

I was lazy and didn't want to define those types. You made me ashamed and I updated the type definitions with proper ones 😄